model/src/main/proto/subtitled_unicode.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Corresponds to a textual value that can be translated to voice.
```

Comment:
```suggestion
// Corresponds to a textual value that can be translated or spoken.
```

This should be clear enough. Technically, we can translate both the textual representation & provide voiceovers for it.